concourse/pipelines/partner-image-validations.jsonnet

```diff
@@ -338,6 +351,16 @@ local ubuntuproposedimages = [
       },
     }
     for family in ubuntudevelimages
+  ] + [
```

Comment:
this block can similarly be merged by making line 353 above into 'ubuntudevelimages + ubuntuarm64develimages'

Reply:
Done.